router/image/templates/nginx.conf

```diff
@@ -15,43 +15,47 @@ http {
     sendfile on;
     tcp_nopush on;
     tcp_nodelay on;
-
     # The Timeout value must be greater than the front facing load balancers timeout value.
     # Default is the deis recommended timeout value for ELB - 1200 seconds + 100s extra.
-    {{ $defaultTimeout := or (.deis_router_defaultTimeout) "1300" }}
-    keepalive_timeout {{ $defaultTimeout }};
+    keepalive_timeout "{{ if exists "/deis/router/defaultTimeout" }}{{ getv "/deis/router/defaultTimeout" }}{{ else }}1300{{ end }}";
```

Comment:
does quoting the keepalive timeout make any changes to the value? We should probably leave it unquoted for feature parity